src/services/subquery/explorer.ts

```diff
@@ -135,18 +135,30 @@ export default class SubqueryExplorer implements Explorer {
     }
   }
 
-  public async getAccountReferralRewards(referrer?: string): Promise<any> {
+  /**
+   * Get Referral Rewards summarized by referral
+   */
+  public async getAccountReferralRewards(referrer?: string): Promise<Nullable<ReferrerRewards>> {
+    const rewardsInfo: ReferrerRewards = {
+      rewards: FPNumber.ZERO,
+      invitedUserRewards: {},
+    };
+
+    let after = '';
+    let hasNextPage = true;
+
     try {
-      const params = {
-        filter: referrerRewardsFilter(referrer),
-      };
-      const { referrerRewards } = await this.request(ReferrerRewardsQuery, params);
-      const rewardsInfo: ReferrerRewards = {
-        rewards: FPNumber.ZERO,
-        invitedUserRewards: {},
-      };
-      if (referrerRewards) {
-        (referrerRewards.nodes as Array<ReferrerReward>).forEach((item) => {
+      do {
+        const response = await this.getAccountlRewards(referrer, after);
```

Comment:
could you please fix typo getAccount`l`Rewards ?

Reply:
Yes, sure!!! Thank you!